components/registerComponents/RegistrationQuestion.tsx

```diff
@@ -89,35 +89,48 @@ function Question(props) {
         </TextField>
         <ErrorMessage
           name={props.question.name}
-          render={(msg) => <div className="text-red-600">{msg}</div>}
+          render={(msg) => <div className="text-red-600 poppins-regular">{msg}</div>}
         />
       </Fragment>
     );
   } else if (props.type === 'checkbox') {
     return (
       <Fragment>
-        <label htmlFor={props.question.name} className="mt-4">
+        <label htmlFor={props.question.name} className="mt-4 poppins-regular mb-1">
           {props.question.required ? '*' : ''}
           {props.question.question}
         </label>
         <div role="group" aria-labelledby="checkbox-group" className="flex flex-col">
           {props.question.options.map((option) => (
-            <label key={option.value}>
-              <Field type="checkbox" name={props.question.name} value={option.value} />
+            <label
+              key={option.value}
+              className={`text-[#313131] text-sm ml-2 ${
+                props.question.id === 'codeOfConduct' ? 'poppins-semibold' : 'poppins-regular'
+              }`}
+            >
+              <Field
+                type="checkbox"
+                name={props.question.name}
+                value={option.value}
+                className="rounded border-[#313131] border-2 cursor-pointer"
+              />
               &nbsp;{option.title}
+              <a href={option.link} className="text-[#40B7BA] hover:underline">
+                {option.linkText}
+              </a>
```

Comment:
Thoughts on making `link` and `linkText` optional fields and then only render the anchor if `option.link` and `option.linkText` is not `undefined`?